test/WebSites/TagHelpersWebSite/TagHelpers/AddProcessedAttributeTagHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNet.Razor.Runtime.TagHelpers;
+
+namespace TagHelpersWebSite.TagHelpers
+{
+    [TargetElement("input")]
+    public class AddProcessedAttributeTagHelper : TagHelper
```

Comment:
I don't see what would be lost if this tag helper were removed. See that the minimized `processed` attribute shows up in the generated HTML. But how does that relate to the rest of this PR?